content/chainguard/chainguard-images/working-with-images/compiled-programs/glibc-vs-musl.md

```diff
@@ -0,0 +1,328 @@
+
+> **Note**: Several sections of this guide present data about the differences between glibc and musl across various categories. You can recreate some of these examples used to find this data with the Dockerfiles and C program files hosted in the `glibc-vs-musl` directory of the [Chainguard Academy Images Demos repository](https://github.com/chainguard-dev/edu-images-demos/tree/main/glibc-vs-musl).
+
+
+## High-level Differences between glibc and musl
+
+The GNU C Library ([glibc](https://www.gnu.org/software/libc/)), driven by the [GNU Project](https://www.gnu.org/gnu/thegnuproject.en.html), was first released in 1988. glibc aims to provide a consistent interface for developers to help them write software that will work across multiple platforms. Today, glibc has become the default implementation of the C standard library across the majority of Linux distributions, including Ubuntu, Debian, Fedora, and even Chainguard's Wolfi.
+
+[musl](https://musl.libc.org/about.html) (pronounced "muscle") was first released in 2011 as an alternative to glibc. The goal of musl is to strive for "simplicity, resource efficiency, attention to correctness, safety, ease of deployment, and support for UTF-8/multilingual text." While not as widely used as glibc, some Linux distributions are [based on musl](https://wiki.musl-libc.org/projects-using-musl), the most notable being Alpine Linux.
+
+The following table highlights some of the main differences between glibc and musl.
+
+| Criteria        	| glibc                                                      	| musl                                                               	|
+| ------------------- | -------------------------------------------------------------- | ---------------------------------------------------------------------- |
+| First Release   	| 1988                                                       	| 2011                                                               	|
+| License         	| GNU Lesser General Public License (LGPL)                   	| MIT License (more permissive)                                      	|
+| Binary Size     	| Larger Binaries                                            	| Smaller Binaries                                                   	|
+| Runtime Performance | Optimized for performance                                  	| Slower performance                                                 	|
+| Build Performance   | Slower                                                     	| Faster                                                             	|
+| POSIX Compliance	| High, complete implementation                              	| Subset, core functionalities                                       	|
```

Comment:
I apologize, but can we delete the POSIX Compliance row and change to this:

| Compatibility	| POSIX Compliant + GNU Extensions                              	| POSIX Compliant

Reply:
on it!